content/en/docs/rules/versioning.md

```diff
@@ -20,7 +16,7 @@ Element | Description
 
 ### Falco Engine Versioning
 
-The `falco` executable and the `falco_engine` C++ object now support returning a version number. The initial version is 2 (implying that prior versions were 1). We will increment this version any time we make an incompatible change to the rules file format or add new filtercheck fields/operators to Falco.
+The `falco` executable and the `falco_engine` C++ object now support returning a version number. The initial version is 2 (implying that prior versions were 1). We will increment this version any time we make an incompatible change to the rules file format or add new filtercheck fields/operators to Falco. You can check the Falco engine version that your installation supports by running `falco --version`.
```

Comment:
```suggestion
The `falco` executable and the `falco_engine` C++ object now supports returning a version number. The initial version is 2 (implying that prior versions were 1). We will increment this version whenever we make an incompatible change to the rules file format or add new filtercheck fields/operators to Falco. You can check the Falco engine version that your installation supports by running `falco --version`.
```
Should "filtercheck" be in a code fence like the command? (is it, or does it need to be in the glossary?)

Reply:
Great point, I'll update and remove the "filtercheck" wording from here because it's not the right place.

Reply:
Done!